src/components/pages/MatchUp/MatchUpContainer.js

```diff
@@ -4,12 +4,8 @@ import { useOktaAuth } from '@okta/okta-react';
 import RenderMatchUp from './RenderMatchUp';
 import { connect } from 'react-redux';
 
-import { faceoffs, child, votes } from '../../../state/actions';
-import {
-  getChildSquad,
-  getFaceoffsForMatchup,
-  getFaceoffsForVoting,
-} from '../../../api';
+import { squad, child } from '../../../state/actions';
```

Comment:
this is definitely gonna lead to merge conflict, I believe my code uses some of the code you deleted